@xen-orchestra/backups/writers/DeltaBackupWriter.js

```diff
@@ -235,11 +236,9 @@ exports.DeltaBackupWriter = class DeltaBackupWriter extends MixinBackupWriter(Ab
           })
         })
       )
-      return {
-        size: Object.values(sizeContainers).reduce((sum, { size }) => sum + size, 0),
-      }
+      return transferSize
```

Comment:
You are changing the task result: https://github.com/vatesfr/xen-orchestra/blob/master/%40xen-orchestra/backups/docs/VM%20backups/README.md#during-backup